pandas-stubs/core/resample.pyi

```diff
@@ -41,7 +41,10 @@ _FrameGroupByFuncArgs: TypeAlias = (
 )
 
 _SeriesGroupByFunc: TypeAlias = (
-    Callable[[Series], Scalar] | Callable[[Series], Series] | np.ufunc
+    Callable[[Series], Scalar]
+    | Callable[[Series], Series]
+    | Callable[[Series], np.float64]
```

Comment:
you shouldn't need this change because you added `np.float_` to `IndexIterScalar`

Reply:
done